view/base.js

```diff
@@ -434,4 +437,4 @@ function clg( i ) {
 	if ( !window.console )return;
 	var l = arguments.length, o = 0;
 	while ( o < l )console.log(arguments[o++])
-}
\ No newline at end of file
+}
```

Comment:
This change to spacing was accidental (initially) and I was trying to get it back to how you got it, but failed, apparently.  :P